Jenkinsfile_nightly

```diff
@@ -0,0 +1,70 @@
+#!groovy
+
+properties([
+    [
+        $class       : 'GithubProjectProperty',
+        displayName  : 'Expert UI Approve Organisations',
+        projectUrlStr: 'https://github.com/hmcts/rpx-xui-approve-org'
+    ],
+    pipelineTriggers([
+    cron('H 05 * * *')
+    ])
+])
+
+@Library("Infrastructure")
+
+def type      = "nodejs"
+def product   = "xui"
+def component = "webapp"
+def channel   = '#xui-pipeline'
+
+withNightlyPipeline(type, product, component) {
+    enableSlackNotifications(channel)
+
+    enableFullFunctionalTest(30)
+    enableMutationTest()
+
+    env.TEST_URL = 'https://xui-ao-webapp-aat.service.core-compute-aat.internal/'
+    after('checkout') {sh 'yarn cache clean'}
```

Comment:
can we please switch to using YarnBuilder as per the Jenkins_CNP file, this method invocation may break